### DISCUSSION TITLE
Tables should have variable corner rounding

### DISCUSSION THREAD

**Discussion by nzaharov** [saadeghi/daisyui#1840] [Ideas / feature requests]
The `table` class seems to add hardcoded radii for the corners of the table element. This makes it inconsistent (at least in my eyes) with the design languages of some "sharp-edged" themes (e.g. **lofi**).

**Comment by nzaharov** [comment]
Looking at https://github.com/saadeghi/daisyui/commit/4f0c0cf1db80d52a3178d3f2bfaccab50bfd5758 - is my proposal relevant?

**Reply by saadeghi** [reply]
>is my proposal relevant?

Yes, I'm working on it. I can't change these things on version `2.x` because it will break some website which are relying on the current style and behavior. But it will be fixed in version `3.0`
